README.md

```diff
@@ -12,11 +12,13 @@ To build just the vocoder itself you will need:
     - Debian
     - ALSA
 - Hardware
-    - Raspberry Pi 3 (or newer)
+    - Raspberry Pi 3 (or newer) running a 64bit OS
     - Codec Zero
```

Comment:
I think the hardware isn't necessarily a codec zero, just that the audio comes in on the correct channels and alsa sets the inputs and outputs we need.

I don't think we ever talk about setting up the codec zero anywyhere either

Reply:
However, this is written to work on a codec zero. Use something else at your own risk.

Reply:
It won't work on other hardware without changing the alsa setup, so I'd say the codec zero is required. 

You're right about the setup though, however, we don't know exactly what the final setup will be at the moment, so I'd say it's better to leave it out than to have a conspicuous section.

Reply:
Ok, it can be left for now